pages/faqs.md

```diff
@@ -53,16 +53,56 @@ If your command-line client was installed manually, please check the path of the
+
+## Improving Exercism
+
+### This is great! How do I get involved?
+There are a few different ways - becoming a mentor, managing a language track or reporting (or addressing!) issues on GitHub. You can see more on the [How to Contribute page](https://exercism.io/contribute). Version 3 of Exercism is also being built right now and needs help from track maintainers. The [v3 repository](https://github.com/exercism/v3) has more information.
 
+### How do new language tracks get added to the site?
+A new language track gets created when a member of the community takes the lead on it and becomes a maintainer of the track. If you'd like to get involved in helping set one up, there are [instructions here](https://github.com/exercism/request-new-language-track/blob/master/README.md).
+
+### Why are issues taking a while to get resolved?
+Version 3 of Exercism is being actively worked on, which means that contributors and maintainers are splitting their time between the current version (V2) and the next.
+
+### What happened to my Team from the old site?
 We have a brand new dedicated Teams website coming soon. We will tweet out as soon as it's launched. Follow us at [@exercism_io](https://twitter.com/exercism_io).
 
+## Opening an Issue
+
+Before submitting an issue, be sure to check the relevant GitHub issue tracker to see if it has already been reported or resolved:
+
+  1. [The Exercism Website or Product](https://github.com/exercism/website-copy/issues)
+  2. [The Command-Line Interface (CLI) Client](https://github.com/exercism/CLI/issues)
+  3. [Exercises](https://github.com/search?q=topic%3Aexercism-track+org%3Aexercism&type=Repositories) - Select your language and then look at the issues tab
+
+You can search through issues (remove the `is:open` filter to include closed/resolved issues). Try a few different keywords.
+
+### What if my issue is not listed here or in GitHub?
+  If your problem hasn't been resolved or reported, then create an issue in the appropriate repository by selecting the green **New issue** button.
+
+Make sure to include the following information:
+  1. The output of the `exercism troubleshoot` command (for CLI issues)
+  1. Details on the issue you are experiencing
+  1. Instructions on how to reproduce the issue
+  1. If applicable, reference to any related issue using its issue number (formatted like #1203)
+
+### What if there is an issue with instructions or language on the website?
+  If you have spotted a typo or if you have a suggestion for clearer language or instructions, then [create an issue for Exercism Website Copy](https://github.com/exercism/website-copy/issues/new) with the following information:

```

Comment:
While the website-copy repo contains text, a lot of the text (including exercise instructions) are actually in a language's repo. And those instructions are in turn based on the data in the [problem-specifications repo](https://github.com/exercism/problem-specifications). So maybe being a little more specific on which language issues goes in what repo could help here?

Reply:
Thanks @ErikSchierboom - I'll update this with additional text once I've looked into it a bit more!

Reply:
Great! Do you need any help with the text?

Reply:
Not just yet, but if you have a moment #1623 needs reviewing - it's for the mentors readme!

Reply:
I've updated this with a link to the tracks search results page - that should give people the right start? If you think that's still not enough (and/or the problem specs repo needs to be linked as well) then would definitely appreciate some more direction. Thanks Erik.

Reply:
Yeah I think that is enough for now. Excellent work!